### ISSUE TITLE
Dependabot not responding after latest merge. No thumbs up after commands either

### ISSUE THREAD

**Issue by planetf1** [dependabot/dependabot-core#5232]
**Package ecosystem**

Maven artefacts - gradle build tool

**Package manager version**

Gradle 7.4.2

**Language version**
Java 11

**Manifest location and content before the Dependabot update**

/build.gradle

https://github.com/odpi/egeria-connector-integration-topic-strimzi/blob/main/build.gradle

**dependabot.yml content**

**Updated dependency**

https://github.com/odpi/egeria-connector-integration-topic-strimzi/blob/main/.github/dependabot.yml

org.odpi.egeria:audit-log-framework

and others.. All 3.7 -> 3.9

**What you expected to see, versus what you actually saw**

I was doing some cleaning up. Noticed some dependabot PRS hadn’t been merged, so I merged a few. This went fine.
However the last one seems stuck. It did hit a conflict, and now will not respond to commands to rebase or recreate 

I also tried close/reopen.

Dependabot should respond to the commands, by adding a thumbs up, and rebasing/recreating

**Native package manager behavior**

n/a

Just a variable update (these are our own artefacts in another repo)

**Images of the diff or a link to the PR, issue, or logs**

https://github.com/odpi/egeria-connector-integration-topic-strimzi/pull/30

**🕹 Bonus points: Smallest manifest that reproduces the issue**

**Comment by jakecoffman** [comment]
Would you try giving dependabot the command once more? We merged a related fix for this late yesterday after the last time you tried.

**Comment by planetf1** [comment]
@jakecoffman I tried just now - so just 3 mins ago. no response yet.

**Comment by jakecoffman** [comment]
I synchronized Dependabot's database with the latest GitHub data. Occasionally data gets out of sync and we see strange things like this happening. Could you try again? Thanks for you patience!

**Comment by planetf1** [comment]
thanks. I'm not seeing any activity on the referenced PR. I tried a 'recreate'.
It's easy to address manually - should we just watch for future issues. Or I'm happy to keep trying if it helps debug an issue :-)

**Comment by huajie2020** [comment]
now done rdy right？ i oso dunno what is this try see。what is。this how to use。

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by mateuscruz-morningstar** [comment]
We're experiencing the same issue with Ruby / Gemfile updates. Dependabot does not respond to `@dependabot recreate|merge|rebase`.

**Comment by mateuscruz-morningstar** [comment]
Seems to be fixed now.

**Comment by jakecoffman** [comment]
@planetf1 Thanks for leaving it up this long, I was able to reproduce the issue internally so you're welcome to fix it manually now.

It appears after the PR was created, Dependabot failed to save an entry to its own database and so the PR isn't recognized by Dependabot as being created by Dependabot. Thus it is ignoring your commands since it doesn't have record of the PR. 

I've opened an issue internally to track and fix this issue.
